### ISSUE TITLE
A page containing a % will be blank

### ISSUE THREAD

**Issue by haedri** [justmiles/go-markdown2confluence#44]
When creating a simple page containing anywhere a % it will be completely blank in confluence

For example if markdown page contains :
```
# Test

Test page is 100% blank now...
```

Then result will be an empty page in confluence, when running in debug mode the content is also empty :

```
---- RENDERED CONTENT START ---------------------------------

---- RENDERED CONTENT END -----------------------------------
```

Escaping the % will provide the same result

Tested using docker version "markdown2confluence v3.3.0"

**Comment by justmiles** [comment]
Thanks for catching this. I was able to reproduce. I'll take a look at the recent changes and see where this was introduced.

**Comment by justmiles** [comment]
This was introduced by a different attempt to fix image uploads. This is fixed in [v3.3.1](https://github.com/justmiles/go-markdown2confluence/releases/tag/v3.3.1)
